composer.json

```diff
@@ -17,14 +17,20 @@
       "email": "patrick@maatwebsite.nl"
     }
   ],
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/laravel-shift/orm.git"
+    }
+  ],
   "require": {
     "php": "^7.4|^8",
     "doctrine/migrations": "^3.4",
     "doctrine/dbal": "^2.10.1|^3",
-    "illuminate/config": "^6.0|^7.0|^8.0|^9.0",
-    "illuminate/contracts": "^6.0|^7.0|^8.0|^9.0",
-    "illuminate/console": "^6.0|^7.0|^8.0|^9.0",
-    "laravel-doctrine/orm": "^1 | ^2@dev"
+    "illuminate/config": "^6.0|^7.0|^8.0|^9.0|^10.0",
+    "illuminate/contracts": "^6.0|^7.0|^8.0|^9.0|^10.0",
+    "illuminate/console": "^6.0|^7.0|^8.0|^9.0|^10.0",
+    "laravel-doctrine/orm": "dev-l10-compatibility"
```

Comment:
That branch will most likely not work as it upgraded to DBAL 3 while not considering all the breaking changes. A PR addressed DBAL 3, and we are now focusing on fixing remaining deprecations before L10. 
You could open a PR for L10 on the 2.x branch in orm though :) 

See https://github.com/laravel-doctrine/orm/issues/542

Reply:
@eigan Thank you! I've opened a PR in laravel-doctrine/orm, and changed the branch name!